src/main/docs/guide/azureKeyVault/distributedConfiguration.adoc

```diff
@@ -0,0 +1,24 @@
+You can leverage https://docs.micronaut.io/latest/guide/index.html#distributedConfiguration[Distributed Configuration] and rely on Azure Key Vault to store your Key/Value secret pairs.
+
+To enable it, add a `src/main/resources/bootstrap.yml` configuration file to your project:
+
+.bootstrap.yml
+[source,yaml]
+----
+micronaut:
+    application:
+        name: hello-world
+    config-client:
+        enabled: true
+    azure:
+      key-vault:
+        vaultUrl: <key_vault_url>
```

Comment:
maybe the example should contain kebab case variables, `vault-url` instead of `vaultUrl` :)